src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -117,9 +118,9 @@ public static float getBreakSpeed(EntityPlayer player, IBlockState state, float
         return (MinecraftForge.EVENT_BUS.post(event) ? -1 : event.newSpeed);
     }
 
-    public static PlayerInteractEvent onPlayerInteract(EntityPlayer player, Action action, World world, BlockPos pos, EnumFacing face)
+    public static PlayerInteractEvent onPlayerInteract(EntityPlayer player, Action action, World world, BlockPos pos, EnumFacing face, Vec3 localPos)
     {
```

Comment:
Would another method that sets it, and then returns itself be better? That way it could still be in a single line for the patches.